app/src/main/java/org/oppia/android/app/player/state/DragDropSortInteractionView.kt

```diff
@@ -45,10 +51,68 @@ class DragDropSortInteractionView @JvmOverloads constructor(
   private lateinit var dataList: List<DragDropInteractionContentViewModel>
   private lateinit var onDragEnd: OnDragEndedListener
   private lateinit var onItemDrag: OnItemDragListener
+  private var itemTouchHelper: ItemTouchHelper? = null
+  companion object {
+    private const val LONG_PRESS_TIMEOUT_MS = 30L
+  }
+
+  private val touchListener = object : OnItemTouchListener {
+    private var startX = 0f
+    private var startY = 0f
+    private val touchSlop = ViewConfiguration.get(context).scaledTouchSlop
+    private var currentTimer: LiveData<Any>? = null
+
+    override fun onInterceptTouchEvent(rv: RecyclerView, e: MotionEvent): Boolean {
+      val fragment = FragmentManager.findFragment<Fragment>(this@DragDropSortInteractionView)
```

Comment:
This is done inside touch events, meaning it’s called frequently. Consider for example, 
```
private val fragment: Fragment by lazy {
    FragmentManager.findFragment<Fragment>(this@DragDropSortInteractionView)
}
```

Reply:
Can use the same variable for `cancelCurrentTimer` below.